src/ui/App.swift

```diff
@@ -50,8 +50,19 @@ class App: NSApplication, NSApplicationDelegate {
         // TODO: undeterministic; events in the queue may still be processing; good enough for now
         DispatchQueue.main.async { () -> () in Windows.sortByLevel() }
         preloadWindows()
+        
+        #if DEBUG
+        showPreferencesPanel()
+        #endif
+        
```

Comment:
If we set `hide menubar icon` to `true`, it will do nothing when we debug the app, we won't be able to open the preferences panel, so I think we may just leave this code.